website/src/content/docs/general/support-policy.md

```diff
@@ -1,39 +1,82 @@
 ---
 title: General Helm Support Policy
+sidebar:
+  order: 2
 ---
 
 This manual contains general information on working with TrueCharts Charts using Helm directly.
-As well as when editing Helm Yaml for tools like FluxCD, KubeApps and Rancher
+As well as when editing Helm Yaml for tools like FluxCD.
 
 :::tip
 
 Please, always remember to check the content specific to the chart.
 
 :::
 
-We at TrueCharts offer limited free support via a support-ticketing system on Discord.
-However the scope of this support is limited to two things:
+We at TrueCharts offer free support via a support-ticketing system on Discord.
 
-- ClusterTool Commands
-- The `values.yaml` of our Helm charts
+## General Support Policy
+
+- We help out where we can, as long as you tried to do it yourself first. We do not provide a walkthrough service.
+- We expect basic experience with the Linux/POSIX shell (ie. `cp`, `mv`, `ls`).
```

Comment:
Duplicate of the below prior knowhow section, can we merge those into this section?
Because its just 3 pointers back there...